pycycle/elements/flow_start.py

```diff
@@ -41,9 +45,13 @@ def setup(self):
         set_stat_MN = SetStatic(mode="MN", thermo_data=thermo_data,
                                 init_reacts=elements, fl_name="Fl_O:stat")
 
-        self.add_subsystem('exit_static', set_stat_MN, promotes_inputs=('MN', 'W'),
+        self.add_subsystem('exit_static', set_stat_MN, promotes_inputs=('MN', 'W', 'init_prod_amounts'),
                            promotes_outputs=('Fl_O:stat:*', ))
 
+        if WAR > 0:
+            set_WAR = SetWAR(thermo_data=thermo_data, WAR=WAR, elements=elements)
+            self.add_subsystem('WAR', set_WAR, promotes_outputs=('init_prod_amounts',))
```

Comment:
is the `WAR` component now providing the init_prod_amounts to `exit_static`? if so, you have an execution order issue here. WAR should come first